WBooks/SuggestBook/SuggestBookViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  SuggestBookViewController.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 12/04/2022.
+//
+
+import UIKit
+
+final class SuggestBookViewController: UIViewController, UINavigationControllerDelegate {
+    private let suggestBookView = SuggestBookView()
+    private let imagePickerController = UIImagePickerController()
+    private let suggestBookViewModel: SuggestBookViewProtocol
+    
+    init(suggestBookViewModel: SuggestBookViewModel) {
+        self.suggestBookViewModel = suggestBookViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpNavBar()
+        suggestBookView.onAddNewButton = addImage
+        suggestBookView.onSubmitButton = submitSuggestion
+    }
+    
+    override func loadView() {
+        view = suggestBookView
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        suggestBookView.submitButton.applyGradient()
+    }
+    
+    private func setUpNavBar() {
+        let title = UILabel()
+        title.font = UIFont.boldSystemFont(ofSize: 17)
+        title.textColor = .white
+        title.text = "SUGGEST_BOOK_TITLE".localized()
+        title.backgroundColor = .clear
+        navigationItem.titleView = title
+
+        let navbarAppearance = UINavigationBarAppearance()
+        navbarAppearance.backgroundColor = .backgroundNavBar
+
+        navigationController?.navigationBar.isTranslucent = false
+        navigationController?.navigationBar.standardAppearance = navbarAppearance
+        navigationController?.navigationBar.scrollEdgeAppearance = navbarAppearance
+    }
+    
+    func addImage() {
+        chooseImageUbicationAlert()
+    }
+    
+    func submitSuggestion() {
+        if suggestBookView.checkFieldsNotEmpty() {
+            suggestBookViewModel.getParametersBook(author: suggestBookView.bookAuthor.text!,
+                                  title: suggestBookView.bookName.text!,
+                                  genre: suggestBookView.bookGenre.text!,
+                                  year: suggestBookView.bookYear.text!)
+            suggestBookViewModel.suggestBook { error in
```

Comment:
This can be 
let book = Book(author, title, genre, year)
suggestBookViewModel.suggestBook(book) { ...